src/main/app/src/Components/LandscapeComponent/Landscape/Landscape.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect, useContext, ReactElement } from 'react';
+
+import { ReactSvgPanZoomLoader, SvgLoaderSelectElement } from 'react-svg-pan-zoom-loader';
+import { ReactSVGPanZoom, TOOL_AUTO, Tool, Value } from 'react-svg-pan-zoom';
+import { useParams } from 'react-router-dom';
+
+import LandscapeItem from '../Item/LandscapeItem';
+import Command from '../../CommandComponent/Command';
+import GenericModal from '../../ModalComponent/GenericModal';
+import LandscapeContext from '../../../Context/Landscape.context';
+import CommandContext from '../../../Context/Command.context';
+
+import './Landscape.scss';
+import { ILandscape } from '../../../interfaces';
+
+const Landscape: React.FC = () => {
+  const [tool, setTool] = useState<Tool>(TOOL_AUTO);
+
+  // It wants a value or null but if we defined it as null it throws an error that shouldn't use null
+  // In their own documentation, they initialize it with {}, but that will invoke a typescript error
+  // @ts-ignore
+  const [value, setValue] = useState<Value>({});
+  const [modalContent, setModalContent] = useState<string | ReactElement | null>(null);
+  const [landscape, setLandscape] = useState<ILandscape | null>(null);
+  const [reloadLandscape, setReloadLandscape] = useState<boolean>(false);
+
+  const landscapeContext = useContext(LandscapeContext);
+  const commandContext = useContext(CommandContext);
+  const { identifier } = useParams();
+
+  useEffect(() => {
+    const index = landscapeContext.landscapes.findIndex(i => i.identifier === identifier);
+    setLandscape(landscapeContext.landscapes[index]);
```

Comment:
Looks like ``find()`` could be used here. But that's not a big deal here.

I would at least rename ``i``: Sound like a name for an index, but it contains the value of the array. ``landscape`` would be much better here.